packages/create-cloudflare/src/helpers/__tests__/args.test.ts

```diff
@@ -112,17 +112,18 @@ describe("Cli", () => {
 			]);
 		});
 
-		const stringArgs = [
-			"--framework",
-			"--template",
-			"--type",
-			"--existing-script",
-		];
+		const stringArgs = ["framework", "template", "type", "existing-script"];
 		test.each(stringArgs)("%s requires an argument", async (arg) => {
-			await expect(parseArgs(["my-react-project", arg])).resolves.toEqual({
+			const logSpy = vi.spyOn(console, "error").mockImplementation(() => {});
```

Comment:
This just removes annoying console logs from the test output and also checks we are outputting what is expected.